packages/cli/package.json

```diff
@@ -113,6 +114,7 @@
     "express": "^4.18.2",
     "figures": "^3.2.0",
     "fs-extra": "^9.1.0",
+    "inquirer": "^7.3.3",
```

Comment:
new version of inquirer is 9.2.x, can we use new version?

Reply:
New version causes the flicks, so we need to downgrade the version to 7.x to fix this issue.